autoload/neomake/makers/ft/bib.vim

```diff
@@ -0,0 +1,15 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#bib#EnabledMakers() abort
+    return []
+endfunction
+
+function! neomake#makers#ft#bib#bibtex() abort
+    return {
+                \ 'exe': 'bibtex',
+                \ 'args': ['-terse', '%:r'],
+                \ 'append_file': 0,
+                \ 'errorformat': '%E%m---line %l of file %f',
+                \ 'cwd': '%:p:h'
```

Comment:
Is there a way to find a project's root?  Does it make sense?

Reply:
There is no way to let bibtex understand where he's working as far as I know, and it needs to be working in the exact same directory as the .bib file, otherwise it might not work (something about a parameter named openout_any which most distributions set to "paranoid" in order to prevent bibtex and other tools to overwrite files in unrelated directories).